commit/plugin.go

```diff
@@ -79,6 +80,18 @@ func NewPlugin(
 		offchainCfg.MaxMerkleTreeSize = merklemulti.MaxNumberTreeLeaves
 	}
 
+	if offchainCfg.RMNEnabled {
+		if err := rmnHomeReader.Ready(); err == nil {
+			ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+			defer cancel()
+			if err := rmnHomeReader.Start(ctx); err != nil {
+				lggr.Errorw("Failed to start RMNHome reader", "err", err)
+			}
+		} else {
+			lggr.Errorw("Failed to initialize RMNHome reader", "err", err)
```

Comment:
If RMNHome reader isn't working and RMN is enabled, should we even start the plugin?

Reply:
As discussed offline, I moved the init logic in the factory to let the libocr retry when `NewReportingPlugin` errored